File: app/src/main/java/com/example/sportivo/Reservation.java
package com.example.sportivo;

import java.time.LocalDateTime;
import java.util.Date;

public class Reservation {
    private int reservationId;
    private int userId;
    private int courtId;
    private Court court;
    private Date startTime;
    private Date endTime;

    public Date getStartTime() {
        return startTime;
    }

    public Date getEndTime() {
        return endTime;
    }

    public int getCourtId(){ return courtId; }

    public Court getCourt(){ return court; }

    public int getReservationId(){return reservationId;}
}

File: Sportivo_webAPI/Models/Reservation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Sportivo_webAPI.Models
{
    public class Reservation
    {
        public int ReservationId { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
        public Court Court { get; set; }
        public int CourtId { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }

        public Reservation() { }
        public Reservation(DateTime start, DateTime end, int courtId)
        {
            StartTime = start;
            EndTime = end;
            CourtId = courtId;
        }
    }
}
